From 63bfd2a1b77147f0ab479ee3bdc64dad62424c59 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Tue, 16 Oct 2018 14:52:47 +0200
Subject: [PATCH 0742/1051] phy: add rxaui mode

Add the RXAUI PHY mode to the phy_mode enum to allow configuring generic
PHYs to RXAUI mode by using set_mode callback.

Change-Id: Id77277122b36508dfbcced3137c402431b24fb46
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/60492
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 include/linux/phy/phy.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/phy/phy.h b/include/linux/phy/phy.h
index cd8268a4c358..b09643ed270b 100644
--- a/include/linux/phy/phy.h
+++ b/include/linux/phy/phy.h
@@ -42,6 +42,7 @@ enum phy_mode {
 	PHY_MODE_UFS_HS_B,
 	PHY_MODE_PCIE,
 	PHY_MODE_SATA,
+	PHY_MODE_RXAUI,
 };
 
 /**
-- 
2.17.1

